crates/hdk/src/lib.rs

```diff
@@ -1,40 +1,31 @@
 //! The Holochain Development Kit (HDK) provides high and low level functions for writing Holochain applications.
 //!
-//! Functions of a Holochain application (hApp) can be organized into reusable components. In Holochain terminology these components are called "zomes".
-//! One or multiple zomes are compiled into a WebAssembly (WASM) binary, referred to as a DNA. All of the DNAs of an application are bundled to a hApp.
-//! In short, that structure is __hApp -> DNA -> zome -> function__.
-//!
-//! hApps are required to produce and validate data deterministically. There's a data model and a domain logic part to each hApp. In Holochain, the
-//! data model is defined in integrity zomes and the domain logic is written in coordinator zomes. See Integrity zomes and Coordinator zomes further down and
-//! [Holochain Deterministic Integrity (HDI)](crate::prelude::hdi) for more information.
-//!
-//! Since hApps are run as a binary on the hosting system, they must be sandboxed to prevent execution of insecure commands.
-//! Instead of writing and maintaining a custom format and specification for these artifacts as well as a runtime environment to execute them,
-//! Holochain makes use of WASM as the format of its applications. WASM binaries meet the aforementioned requirements as per the
-//! [WebAssembly specification](https://webassembly.github.io/spec/core).
+//! Holochain is built as a client-server architecture. The Conductor, Holochain's runtime, acts as the server.
+//! Its [Conductor API](https://docs.rs/holochain_conductor_api/latest/holochain_conductor_api) can be queried
+//! by a client to manage hApps and send requests to hApp functions.
```

Comment:
This might be a good place to briefly define a hApp. The linked article can do the heavy lifting, of course. Suggested definition:

```suggestion
//! by a client to manage hApps and send requests to hApp functions.
//! A hApp is a bundle of WebAssembly modules that are hosted in the Conductor's
//! WebAssembly VM. They define callbacks and API functions that the Conductor can
//! access, and they can access functionality that the Conductor exposes to them.
```

Reply:
On further reading, I see you do define things... sometimes nice to have a succinct definition to start with though; it's sort of like spaced repetition training :)